.travis/install_test.sh

```diff
@@ -11,3 +11,5 @@ wget https://github.com/satooshi/php-coveralls/releases/download/v1.0.1/coverall
 chmod u+x "${HOME}/bin/coveralls"
 
 composer update --prefer-dist --no-interaction --prefer-stable ${COMPOSER_FLAGS}
+composer show --latest 'sonata-project/*'
+composer show --latest 'symfony/*'
```

Comment:
This should be removed, it was for debugging purposes I supposed? dev-kit will remove it if you don't